### ISSUE TITLE
use unix newlines

### ISSUE THREAD

**Issue by gemal** [piwik/piwik#11827]
please use linux newlines in your code
or add a file called `.gitattributes` with the following content:
* text=auto

**Comment by mattab** [comment]
I think i'm seeing this same issue after uploading piwik to a server:

```

Errors below may be due to a partial or failed upload of Piwik files.
--> Try to reupload all the Piwik files in BINARY mode. <--

File size mismatch: /var/app/current/vendor/piwik/decompress/libs/PclZip/gnu-lgpl.txt (expected length: 26934, found: 26430)
File size mismatch: /var/app/current/vendor/piwik/decompress/libs/PclZip/readme.txt (expected length: 22011, found: 21590)
```

**Comment by sgiehl** [comment]
that was fixed in #11869 and related PRs
